clang/include/clang/Driver/Driver.h

```diff
@@ -617,7 +617,8 @@ class Driver {
 
   /// getSYCLDeviceTriple - Returns the SYCL device triple for the
   /// specified subarch
```

Comment:
The comment could be updated to mention the additional argument `llvm::opt::Arg *Arg = nullptr` added.